### PR TITLE
IDVA5-668-Update-content-and-title-for-Company-is-not-active-screen

### PR DESCRIPTION
**Pull request by abhiahirv1**

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/main/utils/properties.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { getEnvironmentVariable, getEnvironmentValue } from "./environment/envir
 
 export const APPLICATION_NAME = "acsp-web";
 
-export const ACCOUNT_URL = getEnvironmentVariable("ACCOUNT_URL", "false");
+export const ACCOUNT_URL = getEnvironmentVariable("ACCOUNT_LOCAL_URL", "false");
```
Comment (sampankumar) [inline src/main/utils/properties.ts:5]: What is ACCOUNT_LOCAL_URL?

File: src/main/controllers/features/limited/companyInactiveController.ts [hunk 2]
```diff
@@ -3,21 +3,21 @@ import * as config from "../../../config";
...
+    const company : Company = session?.getExtraData(COMPANY_DETAILS)!;
 
     const lang = selectLang(req.query.lang);
     const locales = getLocalesService();
 
     res.render(config.LIMITED_COMPANY_INACTIVE, {
         previousPage: addLangToUrl(BASE_URL + LIMITED_WHAT_IS_THE_COMPANY_NUMBER, lang),
         startPage: addLangToUrl(BASE_URL, lang),
-        title: "Company is not active",
+        title: "You cannot register this company",
```
Comment (sampankumar) [inline src/main/controllers/features/limited/companyInactiveController.ts:20]: Does this title needs to come from language json file?

File: src/main/utils/properties.ts (hunk 1 above)
Comment (abhiahirv1) [reply src/main/utils/properties.ts:5]: It is to sign out

File: src/main/controllers/features/limited/companyInactiveController.ts (hunk 2 above)
Comment (abhiahirv1) [reply src/main/controllers/features/limited/companyInactiveController.ts:20]: No

**Review by ttingle-ch — APPROVED** [review]

**Review by sardhani-ch — APPROVED** [review]

**Pull request merged**